typescript/packages/well-completions-plot/demo/example-data/well-completions-csv2json.py

```diff
@@ -0,0 +1,464 @@
+import pandas
```

Comment:
This file already exists in /example-data. Can we use the top-level file instead of duplicatiog it ?

Reply:
Not used, so will be deleted